Jint/Native/JsValue.cs

```diff
@@ -279,60 +279,44 @@ public Types Type
         /// <returns></returns>
         public static JsValue FromObject(Engine engine, object value)
         {
+            JsValue result;
+
             if (value == null)
             {
                 return Null;
             }
 
             foreach (var converter in engine.Options._ObjectConverters)
             {
-                JsValue result;
+
                 if (converter.TryConvert(value, out result))
                 {
                     return result;
                 }
             }
 
-            var typeCode = System.Type.GetTypeCode(value.GetType());
-            switch (typeCode)
+            Dictionary<Type, Func<object, JsValue>> jsValueConverters = new Dictionary<Type, Func<object, JsValue>>()
```

Comment:
Have you been able to compile yet? If not, you might be missing [DotNetCore tools for VS 2015 ](https://download.microsoft.com/download/5/5/7/557D02A5-C3D0-4EF6-A570-4F75CD0DA5BF/DotNetCore.1.0.1-VS2015Tools.Preview2.0.2.exe)

About Convert.ToTypeCode, I've found no information about it on MSDN :( . I had to replace Type.GetTypeCode with a dictionary because I couldn't find a way with NetStandard 1.0 to get the type code from a type (although I've just noticed that I forgot to refactor the local dictionary to a static field as I intended)

Anyway if you have some method to recover the TypeCode that works with NetStandard 1.0 I'll revert the change and get rid of that ugly dictionary